### DISCUSSION TITLE
Can't pay with card, button not working on homepage. Therefore cant activate orgganization plan

### DISCUSSION THREAD

**Discussion by VilleBio** [opengisch/QField#3948] [Bugs]
I have the problem stated in the subject of this post: I Can't pay with card, button not working. Therefore cant activate organization plan. We had registered our organization before the 22nd in hopes we would get the discount aswell but it seems we are not eligible. Why? 

And we can't even activate the plan. 

We need support. As a paying customer how do we get support on this?

We are out in the field collecting data at the moment and need help to get the plan running asap.

**Comment by GlaDal** [comment]
Hi, It's not exactly the same problem than VilleBio but the idea sounds like the same. I can't paid by card. For that reason, I asked to Marco (with a copy to Hello mail adress) a devis for an organization plan. I don't receive any answer (that's not a problem, I sent the request only two days ago) but I just want to be sure that I will be able to use my projects without any problems and with all the options of the organization plan until I'll receive the devis. 
Thanks and courage for this period of transition... :-)

**Comment by lucienicolier** [comment]
Hi,

Can you send me your organizations name, so I can have a look.

**Reply by VilleBio** [reply]
Yes! It is "EnviroPlanning_organisation". Thank you for the Quick answer

**Reply by VilleBio** [reply]
Payment works now when paying from the invoice pdf. However we have a double invoice because of some fault and didnt recieve the 40% off.

**Reply by GlaDal** [reply]
Hi, Organizanition name is 'Sopreco', my account is 'pierre'. Thanks
